packages/components/src/StoryFeed/Story.tsx

```diff
@@ -13,11 +12,11 @@ export const Story: React.FunctionComponent<StoryProps> = props => {
     <>
       <StoryLink onClick={props.handleOpenStory}>
         <StoryLinkLeft>
-          <StoryTitle>{props.title}</StoryTitle>
-          <TimeStamp>{props.timeStamp}</TimeStamp>
+          <StoryTitle>{props.openGraphData.title}</StoryTitle>
+          <TimeStamp>{props.openGraphData.article.published_time}</TimeStamp>
```

Comment:
I don't think we should use the open graph published time here, we should use the post created-at time

Reply:
@nickreynolds Is post created at the time of link submission or the time the article was published?

Reply:
there's a conversation in slack pod-pulse about which we should use